Jenkinsfile

```diff
@@ -15,7 +15,7 @@ def mavenEnv(Map params = [:], Closure body) {
                     body()
                 }
             }
-            if (junit(testResults: '**/target/*-reports/TEST-*.xml').failCount > 0) {
+            if (junit(testResults: '**/target/surefire-reports/TEST-*.xml,**/target/failsafe-reports/TEST-*.xml').failCount > 0) {
```

Comment:
Being more explicit that these are the only two supported test runners (for example, we do not support TestNG), and also matching the CloudBees-internal version of this code for consistency.